packages/core/src/awsService/appBuilder/serverlessLand/wizard.ts

```diff
@@ -0,0 +1,181 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as AWS from '@aws-sdk/types'
+import * as vscode from 'vscode'
+import { Wizard } from '../../../shared/wizards/wizard'
+import * as path from 'path'
+import { createInputBox } from '../../../shared/ui/inputPrompter'
+import { createCommonButtons } from '../../../shared/ui/buttons'
+import { createQuickPick } from '../../../shared/ui/pickerPrompter'
+import { createFolderPrompt } from '../../../shared/ui/common/location'
+import { createExitPrompter } from '../../../shared/ui/common/exitPrompter'
+import { MetadataManager } from './metadataManager'
+import { ToolkitError } from '../../../shared/errors'
+
+export interface CreateServerlessLandWizardForm {
+    name: string
+    location: vscode.Uri
+    pattern: string
+    runtime: string
+    iac: string
+}
+
+/**
+ * Wizard for creating Serverless Land applications
+ * Guides users through the project creation process
+ */
+export class CreateServerlessLandWizard extends Wizard<CreateServerlessLandWizardForm> {
+    private metadataManager: MetadataManager
+
+    public constructor(context: { defaultRegion?: string; credentials?: AWS.Credentials }) {
+        super({
+            exitPrompterProvider: createExitPrompter,
+        })
+        this.metadataManager = MetadataManager.getInstance()
+    }
+
+    public override async run(): Promise<CreateServerlessLandWizardForm | undefined> {
+        try {
+            // Load metadata from JSON file
+            const projectRoot = path.resolve(__dirname, '../../../../../')
```

Comment:
just to double check, is the metadata.json getting copied to the correct location when you build it? Usually you would need modify copyFiles to move it IIRC

Reply:
That is something I'm currently working on. Will correct that in next PR.